Banking/src/main/java/com/example/Banking/exception/ErrorHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.example.Banking.exception;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+
```

Comment:
Shouldn't be space between class and its corresponding annotation

Reply:
I have removed the space